newIDE/electron-app/app/ModalWindow.js

```diff
@@ -60,6 +70,22 @@ const loadModalWindow = ({
     devTools
   });
 
+  //Prevent any navigation inside the modal window.
+  modalWindow.webContents.on("will-navigate", (e, url) => {
```

Comment:
👍

Reply:
There was some silly code in there first that allowed the user to load the same window multiple times. I might remove the new mute sound on load option if I figure out how to resolve this:
https://github.com/ttencate/jfxr/issues/37

Reply:
Ok he fixed it,so I removed the mute workaround :)